src/Websites/Websites/Cmdlets/AccessRestriction/UpdateAzureWebAppAccessRestrictionConfig.cs

```diff
@@ -0,0 +1,99 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+
+using Microsoft.Azure.Commands.WebApps.Models;
+using System.Management.Automation;
+using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+using Microsoft.Azure.Management.WebSites.Models;
+
+namespace Microsoft.Azure.Commands.WebApps.Cmdlets.WebApps
+{
+    /// <summary>
+    /// this commandlet will let you update the access restriction settings of an Azure Website
+    /// </summary>
+    [Cmdlet("Update", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "WebAppAccessRestrictionConfig", SupportsShouldProcess = true)]
```

Comment:
This cmdlet is causing your analysis failure, you need to have a DefaultParameyterSetName if there is mroe than one parameter set